moveit_ros/moveit_servo/include/moveit_servo/servo_calcs.h

```diff
@@ -252,13 +251,13 @@ class ServoCalcs
 
   // Status
   StatusCode status_ = StatusCode::NO_WARNING;
-  bool stop_requested_ = false;
-  bool paused_ = false;
   bool twist_command_is_stale_ = false;
   bool joint_command_is_stale_ = false;
   bool ok_to_publish_ = false;
   double collision_velocity_scale_ = 1.0;
 
+  std::atomic_bool paused_{ false };
```

Comment:
This is basically the reason I've was hesitant to approve this pr when it was small.  As far as I could tell this value does not need to be atomic.  As far as I can tell this is written to by one thread and read from the other and there is no RMU (read-modify-update) operations done on it.

Reply:
done!